deployer.sh

```diff
@@ -118,7 +118,52 @@ echo "# Using inventory: "
 cat ${INVENTORY_PATH}
 echo "#######################################################################"
 
+
+if [ "${INSTALL_PROMETHEUS}" == "true" ]; then
+    # We're installing Prometheus, this means we have to connect to all
+    # nodes on the cluster to make sure the iscsi initator name is set correctly
+    # and to collect the initator names so we can create the iscsi LUN.
+    echo "Setting initator names..."
+
+    INAME="iqn.1994-05.com.redhat:${NAME_PREFIX}-master001"
+    INITIATORS="${INAME}"
+    sshpass -p"${ROOT_PASSWORD}" ssh ${SSH_ARGS} "root@${MASTER_HOSTNAME}" "echo InitiatorName=${INAME} > /etc/iscsi/initiatorname.iscsi; systemctl restart iscsi.service"
+
+    NODE_NUMBER=0;
+    for IP in ${INFRA_IPS}; do
+        printf -v NODE_NUMBER_PADDED "%03d" ${NODE_NUMBER}
+        INAME="iqn.1994-05.com.redhat:${NAME_PREFIX}-infra${NODE_NUMBER_PADDED}"
```

Comment:
how about dropping the infra|master|root in favor of one consecutive numbering and collapsing these 3 blocks?
(I'm thinking regardless we should have gave one numbering to all nodes that would make things simpler)

Reply:
I think having meaningful names is worthwhile, makes it easier to see which node is misbehaving (for example - if one node uses too much CPU/memory, you want to know what it does. the name helps)

Reply:
That way i'm thinking maybe we should name our nodes
```
01-master
02-infra
03-infra
04-compute
...
```

Might be outside the scope of this PR but maybe its worth while as it will save us some a few blocks here already

Reply:
I've changed this to use a bash function so less code is duplicated.